index.html

```diff
@@ -76,54 +77,56 @@
         <div class="section" id="section0">
             <div class="section-container">
                 <div class="container">
-                    <h1 class="company-name text-center">KARNATAKA TOURS & SERVICES</h1>
+                    <h1 class="company-name text-center">KARNATAKA TOURS &amp; SERVICES</h1>
                     <h4 class="tagline">A mission of service</h4>
                     <!--Images-->
-                    <div class="col-lg-12">
-                        <div class="row home-images-container">
-                            <div class="col-lg-3 col-md-3 col-sm-3 col-xs-3">
-                                <div class="home-gallery-image place1">
-                                    <div>
-                                        BOAT HOUSE
-                                    </div>
+                    <div class="row home-images-container">
```

Comment:
Remove flex property from `.home-images-container` and  adjust images using bootstrap grids

Reply:
Is it needed? It works right? It should be fine.

Reply:
I don't see an advantage of using bootstrap if flex is used.

Reply:
also, this is bs3 and 4 uses flex. Overriding in small cases doesn't make a big impact. So can we keep it?

Reply:
It might be using it internally. But using it explicitly with bootstrap will not make sense. So remove flex property and correct the bootstrap classes. It should not take much time.

Reply:
Alright.

Reply:
Thanks